### PR TITLE
[ref] update env var namings to match expected

### PR DESCRIPTION
**Pull request by jordanrfrazier**
Updates environment variables names to match known patterns in existing documentation / usage.

### CODE REVIEW

File: evaluation/langchain_trulens_full.ipynb [hunk 1]
```diff
@@ -1,518 +1,518 @@
 {
```
Comment (epinzur) [inline evaluation/langchain_trulens_full.ipynb:1]: can you revert the changes in the `evaluation` folder?

File: ragstack-e2e-tests/pyproject.llamaindex.toml [hunk 2]
```diff
@@ -20,7 +20,7 @@ pillow = "^10.2.0"
 testcontainers = "^3.7.1"
 python-dotenv = "^1.0.1"
 trulens-eval = "^0.21.0"
-nemoguardrails = "^0.7.1"
```
Comment (epinzur) [inline ragstack-e2e-tests/pyproject.llamaindex.toml:23]: did you mean to make this update too?

**Review by epinzur — APPROVED** [review]

File: ragstack-e2e-tests/pyproject.llamaindex.toml (hunk 2 above)
Comment (jordanrfrazier) [reply ragstack-e2e-tests/pyproject.llamaindex.toml:23]: Yeah, this should've been done in a previous PR

File: evaluation/langchain_trulens_full.ipynb (hunk 1 above)
Comment (jordanrfrazier) [reply evaluation/langchain_trulens_full.ipynb]: For the formatting changes, I'm using 
```
"[python]": {
  "editor.defaultFormatter": "ms-python.black-formatter"
}

```

Going to merge as is now since I can't revert those without reverting a bunch of stuff manually. Can normalize on one formatter for the team

Comment (epinzur) [reply evaluation/langchain_trulens_full.ipynb]: i meant reverting all the changes in the evaluation folder, not just formatting. 

nothing in that folder is run in CI or exposed to users at the moment.
